src/lib/plugins/players.js

```diff
@@ -30,4 +30,23 @@ module.exports.player=function(player){
       player.setGameMode(mode);
     }
   });
-};
\ No newline at end of file
+};
+
+module.exports.player = function(player, serv) {
```

Comment:
ah you are exporting .player a second time here, that doesn't work, put the command with the gamemode command up there.